packages/react-hooks/src/useAccountInfo.ts

```diff
@@ -38,6 +41,8 @@ export function useAccountInfo (value: string | null, isContract = false): UseAc
   const { isAddress } = useAddresses();
   const accountInfo = useCall<DeriveAccountInfo>(api.derive.accounts.info as any, [value]);
   const accountFlags = useCall<DeriveAccountFlags>(api.derive.accounts.flags as any, [value]);
+  const nominator = useCall<Nominations>(api.query.staking.nominator, [value]);
```

Comment:
```suggestion
  const nominator = useCall<Nominations>(api.query.staking.nominators, [value]);
```

Ooops, type in the previous adjustment. Sorry.